hr_holidays_compute_days/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Salton Massally (<smassally@idtlabs.sl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Employee Compute Leave Days',
+    'version': '8.0.1.0.0',
+    'category': 'Human Resources',
+    'license': 'AGPL-3',
+    'summary': 'Computes the actual leave days '
+               'considering rest days and public holidays',
+    'author': 'Salton Massally<smassally@idtlabs.sl>, '
```

Comment:
usually here are companies (IDLabs?)
contributors including the mail are in the README. 
Anyway, add space between name and email (Salton Massally <smassally@idtlabs.sl>)